astropy/modeling/functional_models.py

```diff
@@ -610,18 +610,12 @@ class Sersic1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     r_eff = Parameter(default=1)
     n = Parameter(default=4)
-    _gammaincinv = None
 
-    def __init__(self, amplitude=amplitude.default, r_eff=r_eff.default,
-                 n=n.default, **kwargs):
-        try:
-            from scipy.special import gammaincinv
-            self.__class__._gammaincinv = gammaincinv
-        except ValueError:
-            raise ImportError("Sersic1D model requires scipy > 0.11.")
-
-        super(Sersic1D, self).__init__(
-            amplitude=amplitude, r_eff=r_eff, n=n, **kwargs)
+    try:
+        from scipy.special import gammaincinv
+        _gammaincinv = gammaincinv
+    except ValueError:
```

Comment:
If `scipy` is not installed, then it will simply raise an `ImportError`, which is the desired behavior.  Here we catch `ValueError` and re-raise `ImportError` in the case where `scipy` is installed, *but* it is an older version where `gammaincinv` is not available.  Hence, we raise `ImportError` with the descriptive reasoning.

The new tests are marked to be skipped if `scipy` is not installed, so I don't think that is the issue.  The tests are failing because of all these "ImportError: cannot import name fitting" everywhere (even from subpackages that I haven't touched).  I haven't been able to track this down yet and all tests pass locally for me.  Please let me know if you have any ideas!